templates/.vars.tmpl

```diff
@@ -0,0 +1,26 @@
+export SWAPPER_ADDRESS='${address1}'
+export CLOSER_ADDRESS='${address2}'
+export DISTRIBUTOR_ADDRESS='${address3}'
+export DISTRIBUTOR_V2_ADDRESS='${address4}'
+export HARVESTER_ADDRESS='${address5}'
+export REPORTER_ADDRESS='${address6}'
+export REX_BANK_KEEPER_ADDRESS='${address7}'
+export SWAPPER_ADDRESS_KEY='${key1}'
+export CLOSER_ADDRESS_KEY='${key2}'
+export DISTRIBUTOR_ADDRESS_KEY='${key3}'
+export DISTRIBUTOR_V2_ADDRESS_KEY='${key4}'
+export HARVESTER_ADDRESS_KEY='${key5}'
+export REPORTER_ADDRESS_KEY='${key6}'
+export REX_BANK_KEEPER_ADDRESS_KEY='${key7}'
```

Comment:
Let's use something other than addressX and keyX

Reply:
done